workshop/Lab0/README.md

```diff
@@ -1,36 +1,41 @@
-# Lab 0: Get the IBM Cloud Container Service
+# Lab 0: Prequisites
 
+## What you'll need before starting Lab 1:
+1. An IBM Cloud account. [Get your free account here](https://console.bluemix.net/dashboard/apps/).
+2. The IBM Cloud Kubernetes Service. 
```

Comment:
No. 2 sounds a bit funky because people don't "get" the kube service, its just there once they get the IBM Cloud account. Perhaps we should remove it?

Reply:
Okay I can remove it.